src/java/picard/sam/BuildBamIndex.java

```diff
@@ -50,7 +50,16 @@
  * @author Martha Borkan
  */
 @CommandLineProgramProperties(
-        usage = "Generates a BAM index (.bai) file.",
+        usage = "Generates a BAM index (.bai) file.  Required for multiple tools including generating statistics" +

```

Comment:
Would be better to focus on what the index is used for in general. The .bai statistics usage is fairly anecdotal.